Frends.Community.Multipart.SendMultipartRequest/Frends.Community.Multipart.SendMultipartRequest.csproj

```diff
@@ -20,10 +20,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="HttpMultipartParser" Version="6.0.1" />
     <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
Frends supports Newtonsoft.Json 12.0.1 so this needs to be decreased to that version.